src/main/java/winstone/HostConfiguration.java

```diff
@@ -281,9 +281,9 @@ protected File getWebRoot(File requestedWebroot, File warfile) throws IOExceptio
                     // If archive date is newer than unzipped file, overwrite
                     File outFile = new File(unzippedDir, elemName);
 
-                    // Fix Zip Slip vulnerability
+                    // Protect against Zip Slip attack
```

Comment:
This suggests there's a real attack possible when that's already slightly misleading.

```suggestion
                    // Disallow unzipping files outside the target dir)
```